src/Roslyn.Diagnostics.Analyzers/Core/RelaxTestNamingSuppressor.cs

```diff
@@ -27,7 +27,9 @@ public sealed class RelaxTestNamingSuppressor : DiagnosticSuppressor
 
         public override void ReportSuppressions(SuppressionAnalysisContext context)
         {
-            if (context.Compilation.GetOrCreateTypeByMetadataName(WellKnownTypeNames.XunitFactAttribute) is not { } factAttribute)
+            context.Compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.XunitFactAttribute, out var factAttribute);
+            context.Compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.BenchmarkDotNetAttributesBenchmarkAttribute, out var benchmarkAttribute);
```

Comment:
Shall it be moved to a separate PR?

Reply:
➡️ This one _could_ be moved to a different PR, but this non-production package is the analyzers specifically created for dealing with scenarios like this.

Reply:
@jmarolf do you have an order preference here?

Reply:
I am fine with this change being a part of this PR

Reply:
Thanks, I'll revert 07c9509 as part of the analyzer upgrade after the signed build is published.